src/monsters.cpp

```diff
@@ -1385,6 +1385,11 @@ bool Monsters::loadLootItem(const pugi::xml_node& node, LootBlock& lootBlock)
 	}
 
 	if ((attr = node.attribute("countmax"))) {
+		int32_t lootCountmax = pugi::cast<int32_t>(attr.value());
+		if (lootCountmax > 100) {
```

Comment:
what about negative value? 🌞

Reply:
There is no need to handle negative values ​​if the value is extracted and passed through the function:
```c++
std::max<int32_t>(1, pugi::cast<int32_t>(attr.value()));
```
first.

Reply:
Then why not use `std::clamp` and do not warn at all? I don't see a reason to warn about too high value, but not warn about too low value. It should be either both or none at all. 🤷

Reply:
A warning could be useful although it's quite unlikely that someone will try to put in a negative value for loot.

Reply:
something like this?
```
		if (lootCountMax > 100 || lootCountMax < 1) {
			std::cout << "[Warning - Monsters::loadMonster] Value for \"countmax\" "<< lootCountMax <<" out of bounds. " << std::endl;
			return false;
		}
```